File: server/nodejs/src/utils/emails/sendVerificationEmail.ts
import { sendEmail } from "../helpers";

export const sendVerificationEmail = async (to: string, token: string) => {
  const confirmLink = `http://localhost:4000/api/auth/new-verification/${token}`;
  const text = `Please click on the following link to reset your password: ${confirmLink}`;
  const html = `<p>Click <a href="${confirmLink}">here</a> to verify your email</p>`;

  await sendEmail({
    to,
    subject: "Confirm your email",
    text,
    html,
  });
};

File: server/python/app/utils/emails/send_verification_email.py
from ..helpers import send_email


def send_verification_email(email, token):
    resetLink = f"http://localhost:4000/api/auth/new-verification/${token}"
    text = f"Please click on the following link to verify your email: ${resetLink}"
    html = f"<p>Click <a href=${resetLink}>here</a> to verify your email</p>"
    send_email(
        email,
        "Password Reset Request",
        "emails/password_reset",
        token=token,
    )
